torch/_prims/__init__.py

```diff
@@ -1702,7 +1704,7 @@ def _squeeze_aten(a: Tensor, dimensions: Sequence) -> Tensor:
   """
 
 squeeze = _make_prim(
-    schema="squeeze(Tensor(a) a, int[] dimensions) -> Tensor(a)",
+    schema="squeeze(Tensor(a) a, SymInt[] original_shape, int[] dimensions) -> Tensor(a)",
```

Comment:
What would it take to change the nvfuser execution stack to record the original shape? If we're designing a new, minimal schema set from scratch, I don't think it's great to make changes to the schemas because of limitations in an execution stack. 

This seems like metadata you could record in the nvfuser execution mode, or many other routes to getting the original shape.

thoughts, others ? cc @ngimel @ezyang @Chillee

Reply:
Metadata is already recorded in `node.meta["tensor_meta"]` so this change is indeed unnecessary.